### DISCUSSION TITLE
推荐里面的图片无法显示，进详情可以显示

### DISCUSSION THREAD

**Discussion by iamckun** [jxxghp/MoviePilot#979] [问题咨询]
![image](https://github.com/jxxghp/MoviePilot/assets/65990499/1adbd07d-f7af-46f3-ab0e-f8b112430fd4)
有一些图片无法显示，只能显示几个类型的。进详情又可以看到。有魔法，这是什么情况啊

**Comment by cng10001** [comment]
![image](https://github.com/jxxghp/MoviePilot/assets/115156545/09c93606-a9be-4bd1-a074-d68f3a7fc9d9)
正在热映那行是豆瓣数据，应该跟魔法没关系吧

**Reply by iamckun** [reply]
需要怎么操作呢，我现在一些显示一些不显示

**Reply by cng10001** [reply]
不懂哦，清下缓存？关了魔法？重启试试？

**Reply by iamckun** [reply]
魔法没看到豆瓣的请求哎
